lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -957,20 +952,22 @@ def test_order_by_query(self, http_query, cc_query):
             "user_id": [unicode(self.user.id)],
             "course_id": [unicode(self.course.id)],
             "sort_key": [cc_query],
-            "sort_order": ["desc"],
             "page": ["1"],
             "per_page": ["11"],
         })
 
-    @ddt.data("asc", "desc")
-    def test_order_direction_query(self, http_query):
+    def test_order_direction(self):
```

Comment:
@cahrens @dianakhuang: I thought there was an invalid test, but it turned out to be in the test_forms.py.  In addition to adding this test, I found a couple of other minor places requiring cleanup that I had missed.  Let me know when you've approved this final commit.  Thanks.